lib/ui/compositing.dart

```diff
@@ -272,10 +272,10 @@ class SceneHost extends NativeFieldWrapperClass2 {
   /// to the child.
   ///
   /// The scene host takes ownership of the provided export token handle.
-  SceneHost(int export_token_handle) {
+  SceneHost(dynamic export_token_handle) {
```

Comment:
We should document that the expected type is a Handle even if we're unable to directly refer to that type here due to platform dependencies.

That said, I wonder whether the platform dependency should be handled some other way.  Among other things, we could wrap up the handle into an "ExportToken" class in Flutter.  That at least provides a point where we can attach documentation even if the handle it contains is a platform dependent resource that ends up having its type erased.